curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/write-reusable-javascript-with-functions.english.md

```diff
@@ -27,7 +27,7 @@ tests:
   - text: <code>reusableFunction</code> should be a function
     testString: assert(typeof reusableFunction === 'function', '<code>reusableFunction</code> should be a function');
   - text: <code>reusableFunction</code> should output "Hi World" to the dev console
-    testString: assert("Hi World" === logOutput, '<code>reusableFunction</code> should output "Hi World" to the dev console');
+    testString: assert(hiWorldWasLogged, '<code>reusableFunction</code> should output "Hi World" to the dev console');
   - text: Call <code>reusableFunction</code> after you define it
     testString: assert(/^\s*reusableFunction\(\)\s*;/m.test(code), 'Call <code>reusableFunction</code> after you define it');
```

Comment:
```suggestion
    testString: assert(/^\s*reusableFunction\(\)\s*/m.test(code), 'Call <code>reusableFunction</code> after you define it');
```
The function can be called without the `;`

Reply:
But all the others are using it, so why introduce an inconsistency?

Reply:
@lynxlynxlynx Not all the others.  On many occasions we have been allowing the user to not include the semi-colon (where it is applicable not to). In the examples we show "best practice", but the tests should allow valid syntax.

Reply:
All others in this file, of course. However I see I misread the diff in the email, so forget I mentioned anything.